background/app/main_test.py

```diff
@@ -11,3 +11,83 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import json
+import main
+import pytest
+
+
+class MockFirestoreClient():
```

Comment:
Guidance I've seen for samples is to use the real service for testing.

Reply:
I'll look into this and see what I can do. I hadn't seen that, and figured that the point here wasn't to test the APIs or libraries, just the sample code. I'll have to set up a project or find out which one is already being used.

Reply:
Tests use the real service for testing now.